### PR TITLE
DEV-2425: web video playback on OSX

### PR DESCRIPTION
**Pull request by jherico**
[DEV-2425](https://highfidelity.atlassian.net/browse/DEV-2425): This uses a new version of Qt that patches a bug related to video playback for web pages.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22048/HighFidelity-Beta-PR16385-6ef6732.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22048/HighFidelity-Quest-Beta-PR16385-6ef6732.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22048/HighFidelity-Beta-PR16385-6ef6732.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22048/HighFidelity-Beta-Interface-PR16385-6ef6732.exe

**Comment by jherico** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22055/HighFidelity-Beta-PR16385-5bf2365.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22055/HighFidelity-Beta-Interface-PR16385-5bf2365.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22055/HighFidelity-Beta-PR16385-5bf2365.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22055/HighFidelity-Quest-Beta-PR16385-5bf2365.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22055/HighFidelity-Beta-PR16385-5bf2365.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22055/HighFidelity-Beta-Interface-PR16385-5bf2365.exe

File: tests-manual/qml/src/main.cpp [hunk 1]
```diff
@@ -205,12 +205,22 @@ void TestWindow::resizeEvent(QResizeEvent* ev) {
...
+    // Deal with some weirdness in the chromium context sharing on Mac.
+    // The primary share context needs to be 3.2, so that the Chromium will
+    // succeed in it's creation of it's command stub contexts.
+    format.setVersion(3, 2);
+    // This appears to resolve the issues with corrupted fonts on OSX.  No
+    // idea why.
+    qputenv("QT_ENABLE_GLYPH_CACHE_WORKAROUND", "true");
+    // https://i.kym-cdn.com/entries/icons/original/000/008/342/ihave.jpg
     QSurfaceFormat::setDefaultFormat(format);
+#endif
```
Comment (AndrewMeadows) [inline tests-manual/qml/src/main.cpp]: There is now no `format.setVersion()` call when not `Q_OS_MAC`, whereas there was one previously.  Should there be an `#elseif` clause here?

Comment (jherico) [reply tests-manual/qml/src/main.cpp]: no, this use case is specifically to deal with the weirdness of OpenGL on mac.

**Pull request merged**